Bottom: d114fd637fe0575ec1d9daef8ff8808cdc43c7b3
Top:    3e4f87735ad887a52f912c34e50df54f44d01781
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-02-25 19:43:45 -0500

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
